File: transmissions/hello.ts
// TRANSMISSION 02 // TYPESCRIPT — types keep the signal clean.

type Signal = { node: string; payload: string; status: "ONLINE" | "OFFLINE" };

const firstTransmission: Signal = {
  node: "00",
  payload: "Hello, World",
  status: "ONLINE",
};

console.log(`[NODE ${firstTransmission.node}] ${firstTransmission.payload}`);

File: transmissions/hello.py
"""TRANSMISSION 01 // PYTHON — where the learning started."""

VITALS = {"status": "ONLINE", "pulse": "STEADY", "mission": "technology x medicine"}


def transmit(signal: str = "Hello, World") -> None:
    print(f"[NODE 00] {signal} :: {VITALS['status']}")


if __name__ == "__main__":
    transmit()
